ospf6d/ospf6_gr_helper.c

```diff
@@ -1373,6 +1373,6 @@ int config_write_ospf6_gr_helper(struct vty *vty, struct ospf6 *ospf6)
 int config_write_ospf6_debug_gr_helper(struct vty *vty)
```

Comment:
I think we need to change this function name as well to avoid confusion in future.